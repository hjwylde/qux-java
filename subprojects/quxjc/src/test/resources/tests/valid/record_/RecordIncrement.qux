void main([str] args):
    xs = {x: 0, y: 1}
    print xs.x
    print xs.y
    xs.x++
    xs.y++
    print xs.x++
    print xs.y
    print xs.x

