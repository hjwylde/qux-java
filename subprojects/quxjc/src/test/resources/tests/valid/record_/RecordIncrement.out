0
1
{x: 2}
1
2
2
{x: 3}
{x: 3, y: 2, z: {x: 3}}
{x: 2, y: 2, z: {x: 3}}
